clang-format.txt

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
You probably didn't intend to add this file, so please remove it :^)

Reply:
Definitely not, sorry ◔̯◔ . I forget that the github client automatically adds all the files I put in the folder